### DISCUSSION TITLE
Import chapters from CUE sheet

### DISCUSSION THREAD

**Discussion by ghost** [yermak/AudioBookConverter#532] [Ideas]
Would be great to be able to do this, InAudible did have this feature, and it was a breeze to fix chapter names and move chapters around using the timecodes in the CUE sheet.

**Comment by vortexae** [comment]
I could also use this feature. Chapter and Verse had this feature too, let you export to/import from CSV. I see that the latest AudioBookConverter has Chapter->Import command, but it doesn't seem to do anything I can see? And no corresponding Export command.

**Reply by ghost** [reply]
I found a set of scripts that can losslessly edit embedded Quicktime chapters in M4B: https://github.com/TheMetalCenter/m4b-mp3-chapters-from-cuesheets

If you know how to install packages you'll be at an advantage. I had no experience, but following the instructions on the main page got it working.

The scripts work by extracting the embedded chapter information into a CUE sheet (via a batch file), then you can edit the CUE sheets to add or change chapters. Afterwards you use another batch file to re-embed them in).

I've just been editing the CUE sheets in a code editor, by converting timecodes from Adobe Audition and pasting them in. Not sure if there is a more efficient way to do this, but TheMetalCenter scripts don't affect the audio quality, so that's the main thing.

**Reply by vortexae** [reply]
Interesting. I'll check it out, but I have no experience with packages either. I was kind of spoiled with editing the chapter break CSV in a text editor and then just importing that into Chapter and Verse via the Import Chapters command, which was comparatively effortless, which is why I'm hoping ABC will have a similarly effortless feature someday in the future. But even if this is a bit more complicated, it looks worth trying out, just to have more tools in my toolset. Thank you!

**Reply by eturk1** [reply]
Maybe remove Chapter/Import from the menu since it is nonfunctional?
